google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -820,10 +829,16 @@ protected SpannerOptions(Builder builder) {
       enableBuiltInMetrics = builder.enableBuiltInMetrics;
     }
     enableEndToEndTracing = builder.enableEndToEndTracing;
-    monitoringHost = builder.monitoringHost;
     defaultTransactionOptions = builder.defaultTransactionOptions;
   }
 
+  private String getResolvedUniverseDomain() {
+    String universeDomain = getUniverseDomain();
+    return (universeDomain == null || universeDomain.isEmpty())
```

Comment:
nit: use `Strings.isNullOrEmpty`. With that, you can also make this a one-liner and skip the variable declaration